pallets/msa/src/lib.rs

```diff
@@ -68,11 +82,35 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::weight(10_000)]
```

Comment:
If the benchmark is separated then we should have a separate benchmark issue for this to make sure it is tracked